tests/derive/occurrences.rs

```diff
@@ -13,7 +13,7 @@ fn test_vec_of_vec() {
         Opt {
             points: vec![vec![1, 2], vec![0, 0]]
         },
-        Opt::try_parse_from(&["test", "-p", "1", "2", "-p", "0", "0"]).unwrap()
+        Opt::try_parse_from(["test", "-p", "1", "2", "-p", "0", "0"]).unwrap()
```

Comment:
Similar to my other comment, are these lints because you are using a newer version of clippy or that CI is not correctly reporting these?

If its just because you are using a new version of clippy, I'd prefer that be made clear and, to be frank, is a lower priority.

Reply:
I'll have to check

Reply:
Can confirm, the reason I got the warning is that I was on `stable`, it is not present on `1.64`.